content/docs/studio/self-hosting/configuration/index.md

```diff
@@ -0,0 +1,59 @@
+# Configuration
+
+Studio uses a standardized, unified YAML configuration file, often referred to
+as `values.yaml` in the documentation.
```

Comment:
Seems really weird to me to talk about this without mentioning Helm. This is not just some "unified YAML" (not sure what that even means). 
Suggest to add a link here somehow: https://helm.sh/docs/chart_template_guide/values_files/

Reply:
We can, of course, mention Helm. My concern here was with overwhelming AMI users with Kubernetes terminology. There are also plans to move to an interactive installer, so manual configuration eventually becomes obsolete for most users.

> This is not just some "unified YAML" (not sure what that even means).

I can see it's confusing, and I'll rephrase it. The point was that the configuration works for both the AMI and Helm flavors.